src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -1651,22 +1651,25 @@ section bridge_edges
 are no longer reachable from one another. -/
 def is_bridge (G : simple_graph V) (e : sym2 V) : Prop :=
 e ∈ G.edge_set ∧
-sym2.lift ⟨λ v w, ¬ (G.delete_edges {e}).reachable v w, by simp [reachable_comm]⟩ e
```

Comment:
What is the motivation for avoiding `delete_edges` here and below?

Reply:
@kmill is thinking of "deprecating" `delete_edges` in favour of always using `from_edge_set`; I think the idea is that this allows using simp lemmas from the lattice structure of sets/graphs and we thus need fewer specialized lemmas.
I wanted to extend kmill's work on trees by proving their characterizations as minimal connected/maximal acyclic graphs ([done here](https://github.com/leanprover-community/mathlib/compare/master...bottine/simple_graph/trees_min_max2)), and for this I needed `add_edges` mirroring `delete_edges`.
So the idea here instead of adding `add_edges` was to refactor so that everything is now done in terms of `from_edge_set`.

Reply:
I see. I'd like to give @kmill a chance to comment on this but I don't want to hold you up too long.

Please ping me here if we don't hear from him within six hours, say.

Reply:
I think he was on holiday, and this is not the most pressing PR for me, so no hurry from my side!

Reply:
I need this PR for LeanCamCombi, but I can wait six hours!

Reply:
Indeed he does seem to be on holiday. OK let's just merge this.